Cargo.toml

```diff
@@ -34,15 +34,15 @@ description = "Boa is a Javascript lexer, parser and compiler written in Rust. C
 [workspace.dependencies]
 
 # Repo Crates
-boa_ast = { version = "~0.21.0", path = "core/ast" }
-boa_engine = { version = "~0.21.0", path = "core/engine" }
-boa_gc = { version = "~0.21.0", path = "core/gc" }
-boa_icu_provider = { version = "~0.21.0", path = "core/icu_provider" }
-boa_interner = { version = "~0.21.0", path = "core/interner" }
-boa_macros = { version = "~0.21.0", path = "core/macros" }
-boa_parser = { version = "~0.21.0", path = "core/parser" }
-boa_runtime = { version = "~0.21.0", path = "core/runtime" }
-boa_string = { version = "~0.21.0", path = "core/string" }
+boa_ast = { version = "~1.0.0", path = "core/ast" }
```

Comment:
Can we list this as "1.0.0-dev" for anyone using the main branch. Once we go to release 1.0, then we can remove the "-dev" suffix.